README.md

```diff
@@ -1,94 +1,274 @@
-- [`./cmd/lukso`](./cmd/lukso): code of LUKSO CLI
-- [`./abis`](./abis) - collection of ABIs from smart contracts that are being interacted with
-- [`./contracts`](./contracts) - collection of said smart contracts
-- [`./contracts/bindings`](./contracts/bindings) - bindings generated from ABIs - to generate new bindings see [Generate bindings](#generate-bindings) section.
-- [`./install`](./install/) - collection of things to support the various installation, signing and notarization requirements.
-- [`./docs`](./docs) Some small content to be inserted into mac pgk file
+```
+tools-lukso-cli
+│
+└───abis                  // Binary Interfaces of LUKSO Smart Contracts
+│
+└───cmd                   // Commands
+│   └───lukso             // LUKSO CLI
+│
+└───contracts             // Solidity Contracts
+│   └───bindings          // Bindings generated from ABIs
+|
+└───docs                  // Updates for Installation Progress
+|
+└───install               // Mandatory Installation Tools
+│   └───cf-wrangler       // Manager for Cloudflare Workers
+│   └───docs-processor    // Markdown to Page Converter
+│   └───macos_packages    // MacOS Codesigning Scripts
+|
+└───pid                   // Process ID Management
+```
+
+## CLI Installation Script
 
-## CLI Installation
 ```sh
-curl https://install.lukso.network | sh
+https://install.lukso.network
 ```
 
-## Downloading and Installing LUKSO
+**Running this script will install the full LUKSO CLI Tool on Mac and Linux.**
+**Installation directory: `usr/local/bin/lukso`**
 
-```bash
-# 1. To install the LUKSO Command Line Interface (CLI)
-$ curl https://install.lukso.network | sh
+## Node Folder Structure
 
-# 2. Create a working folder where you want your clients to store their data
-$ mkdir myLUKSOFolder && cd ./myLUKSOFolder
+> Initializing a LUKSO node will not overwrite existing config, data or keystore folders
 
-# 3. This will initialize your working folder by downloading all network configs from
-# https://github.com/lukso-network/network-configs
-# NOTE: This will not overwrite any existing config, data or keystore folders
-$ lukso init
+```
+lukso-node
+│
+└───configs               // Blockchain Configuration
+|   | config.yaml         // Network Configuration
+|   | genesis.json        // Genesis JSON Data
+|   | genesis.ssz         // Genesis Validator File
+│
+└───data                  // Blockchain Data Storage
+│   └───consensus_data    // Storage of Consensus Client
+│   └───execution_data    // Storage of Execution Client
+│   └───validator_data    // Storage of Validator Client
+│
+└───keystore              // Validator Secrets and Keys
+│   └───keys              // Encrypted Private Keys
+│   └───...               // Folders & Files for Signature Creation
+|   | pubkeys.json        // Validator Public Keys
+|
+└───transaction_wallet    // Validators Transaction Wallet
+|
+| deposit_data.json       // Deposit JSON for Validators
+| node_config.yaml        // Node Configuration File
+```
```

Comment:
This is not entirely true as it will change with current ticket:

https://app.clickup.com/t/861mgjpgt

Esp. the network name will be added: `mainnet-keystore/`, etc.
Also, the configs folder will have subfolders, one per network.

I suggest to remove this part and to add it back in the PR that will be sent with this ticket:
https://app.clickup.com/t/861mgjpgt

Reply:
Thanks, I added it as a blueprint to the internal task and left a WIP